bigquery_etl/schema/__init__.py

```diff
@@ -230,7 +236,11 @@ def _traverse(
                         f"{prefix}.{field_path}",
                         nodes[node_name]["fields"],
                         node["fields"],
-                        update,
+                        update=update,
+                        add_missing_fields=add_missing_fields,
+                        ignore_missing_fields=ignore_missing_fields,
+                        exclude=exclude,
```

Comment:
I'm unclear whether `exclude` is intended to only be for top-level columns.  If so, we wouldn't want to pass it along to the recursive calls.

Reply:
```suggestion
```
Yeah, it was only for top-level columns